plaidml/edsl/edsl_doc.cc

```diff
@@ -0,0 +1,338 @@
+
+// Copyright 2020 Intel Corporation.
+// DO NOT TOUCH THIS FILE
```

Comment:
We can drop this warning if it's no longer a requirement for line numbers to be exactly correct in the docs.